sources/packages/backend/apps/api/src/route-controllers/institution-locations/_tests_/e2e/institution-location.students.controller.getOptionsList.e2e-spec.ts

```diff
@@ -0,0 +1,68 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getStudentToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeDesignationAgreementLocation,
+} from "@sims/test-utils";
+
+describe("InstitutionLocationStudentsController(e2e)-getOptionsList", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get the list of all designated institution location which should contain the only newly created designated location not the newly created non designated location when student requests.", async () => {
```

Comment:
The text is confusing... Should it be something like: "Should get the list of all designated institution location which should contain the only newly created designated location designated location when student requests."?

Reply:
updated as we discussed